en/integrations/plugin-packs/procedures/applications-databases-elasticsearch.md

```diff
@@ -3,47 +3,193 @@ id: applications-databases-elasticsearch
+| total_in_bytes    | Total size of all file stores                         |    B    |
+| docs_count        | Number of documents inside the indice                 |         |
+| size_in_bytes     | Total size of all shards assigned to the node         |    B    |
+
+<!--Rules-->
+
+| Rule name                              | Description                                   |
+| :------------------------------------- | :-------------------------------------------- |
+| App-DB-Elasticsearch-Indice-Statistics | Discover the indices on your Elasticsearch DB |
+| App-DB-Elasticsearch-Node-Statistics   | Discover the nodes on your Elasticsearch DB   |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+Use the discovery module to add the monitoring of your Elasticsearch databases, Go to *Configuration > Services > Scan* to perform a scan.
+
+## Prerequisites 
+
+In order to monitor an Elasticsearch cluster, it must be prepared acccording to Elasticsearch's official documentation: https://www.elastic.co/guide/en/elasticsearch/reference/7.8/monitor-elasticsearch-cluster.html
+In order to be able to communicate with the Centreon poller, the Elasticsearch node's API should use the http protocol and the port 9200.
+
+## Installation
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online Licence Business & IT Editions-->
+
+1. Install the Plugin on all Centreon pollers monitoring Elasticsearch ressources:
+
+```bash
 yum install centreon-plugin-Applications-Databases-Elasticsearch
 ```
 
-## Centreon Configuration
+2. Instal the Plugin-Pack 'Elasticsearch' in the " Configuration  >  Plugin Packs"  page of the Web Centreon interface
+
+<!--Offline Licenses-->
 
-### Create a host using the appropriate template
+1. Install the Plugin on all Centreon pollers monitoring Elasticsearch ressources:
 
-Go to *Configuration \> Hosts* and click *Add*. Then, fill the form as shown by
-the following table:
+```bash
+yum install centreon-plugin-Applications-Databases-Elasticsearch
+```
 
-| Field                   | Value                        |
-| :---------------------- | :--------------------------- |
-| Host name               | *Name of the host*           |
-| Alias                   | *Host description*           |
-| IP                      | *Host IP Address*            |
-| Monitored from          | *Monitoring Poller to use*   |
-| Host Multiple Templates | App-DB-Elascticsearch-custom |
+2. Install the Plugin-Pack RPM on the Centreon server **Central**:
 
-Click on the *Save* button.
+```bash
+yum install centreon-pack-applications-databases-elasticsearch
+```
+
+3. In the *Configuration  \>  Plugin Packs*  page of the Web Centreon interface, install the Plugin-Pack 'Elasticsearch'
+
+<!--END_DOCUSAURUS_CODE_TABS-->
 
 ### Host Macro Configuration
 
-The following macros must be configured on host:
+Apply the "The name of your template" template to your newly created host. Then fill the macros value fileds marked as mandatory below:
```

Comment:
The name of your template ?
In case it should be App-DB-Elasticsearch-custom ;-)